extensions/iceberg/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+description 'Iceberg: Support to read iceberg catalogs.'
+
+ext {
+    icebergVersion = '1.4.3'
+    hadoopVersion = '3.3.6'
+}
+
+dependencies {
+    api project(':engine-api')
+    api project(':engine-stringset')
+    api project(':engine-table')
+
+    implementation project(':engine-table')
+    implementation project(':engine-base')
+    implementation project(':log-factory')
+    implementation project(':Configuration')
+    implementation depTrove3
```

Comment:
I don't think we're using these. We can probably remove them, and should generally audit this file.